modules/core/remove_orphaned.py

```diff
@@ -17,6 +21,9 @@ def __init__(self, qbit_manager):
         self.root_dir = qbit_manager.config.root_dir
         self.orphaned_dir = qbit_manager.config.orphaned_dir
 
+        global _config
+        _config = self.config
+
         self.rem_orphaned()
```

Comment:
```suggestion
        self.pool = Pool(processes=cpu_count()-1)
        self.rem_orphaned()
        self.cleanup_pool()
```
Prefer if we don't use all the cpus when doing processing. Also we can reuse this pool in different areas and then cleanup at the end rather than using context manager

Reply:
Doesn't handle the (rare) case of single core cpu

Reply:
That's true good call. You could do something like
`self.pool = Pool(processes=max(cpu_count()-1,1))`

Reply:
Mind editing the suggestion so I can just accept them all?